html/admin/hlb.css

```diff
@@ -31,14 +31,27 @@ div.lastcolumn {
 }
 
 span.navigation {
-    padding-left: 1em;
-    padding-right: 1em;
+    padding: .5rem 1rem;
     margin-left: 1em;
     margin-right: 1em;
+    display: inline-block;
 }
 
-span.active {
-    background-color: #aaa;
+.navigation{
+    border-bottom: 3px solid rgba(0,0,0,0);
+}
+
+.navigation > a{
+    text-decoration: none;
+    color: #000;
+}
+
+.active{
+    border-color: #1d7491;
+}
+
+.navigation:hover{
+    border-color: #1d7491;
```

Comment:
The header navigation was updated to look similar to the design system styles. In essence, the active item and hovered items are now underlined. I the was having a difficult time trying to find an accessibility way to highlight the active element that also looked decent.